src/main/java/uk/gov/hmcts/dts/fact/repositories/CourtRepository.java

```diff
@@ -16,6 +16,8 @@
     @Query("UPDATE Court c SET c.info = :info, c.infoCy = :infoCy WHERE c.slug in :slugs")
     void updateInfoForSlugs(@Param("slugs") List<String> slugs, @Param("info") String info, @Param("infoCy") String infoCy);
 
+    List<Court> findCourtBySlugStartingWithAndDisplayed(String prefix, boolean active);
```

Comment:
I am confused...where is the SQL query for this method...how does it know how to filter by the prefix?